redfish-core/lib/storage.hpp

```diff
@@ -671,7 +672,8 @@ inline void afterGetSubtreeSystemsStorageDrive(
     asyncResp->res.jsonValue["@odata.type"] = "#Drive.v1_7_0.Drive";
     asyncResp->res.jsonValue["@odata.id"] = boost::urls::format(
         "/redfish/v1/Systems/system/Storage/1/Drives/{}", driveId);
-    asyncResp->res.jsonValue["Name"] = driveId;
+    name_util::getPrettyName(asyncResp, path, drive->second,
+                             "/Name"_json_pointer);
```

Comment:
In addition to this line, I think there seems another line to use `getPrettyName()`.

https://github.com/ibm-openbmc/bmcweb/blob/46c5edc7393e91e629f3cb1e057f69c2c5726a3e/redfish-core/lib/storage.hpp#L882

Reply:
Will address this also as part of new commit (JIRA number will be attached)